core/src/main/gamelogic/Referee.kt

```diff
@@ -0,0 +1,160 @@
+package gamelogic
+
+import gamelogic.map.PoliticalMap
+import gamelogic.occupations.CountryOccupations
+import Country
+import Player
+import PositiveInt
+import gamelogic.combat.Attacker
+import gamelogic.combat.Conqueror
+import gamelogic.combat.resolver.CombatDiceRoller
+import gamelogic.combat.resolver.DiceRollingCombatResolver
+import gamelogic.dice.RandomDie
+import gamelogic.situations.classicCombat.ClassicCombatDiceAmountCalculator
+
+
+//TODO: En lugar de crear esta clase lo que hay que hacer es sacar del ataquer la
+//      responsabilidad de mover las fichas después de atacar. Peeero, no solo hay
+//      que hacer refactor del package combat, sino también de los tests!!! Asi que
+//      queda para otro backlog item.
+class SkipRegroup : Conqueror {
+    override fun armiesToMove(remainingAttackingArmies: PositiveInt): PositiveInt {
+        return PositiveInt(0)
+    }
+
+}
+
+class CountryReinforcement(val country:Country, val n: PositiveInt) {
+    public fun apply(player: Player,occupations: CountryOccupations) {
+        if (occupations.occupierOf(country) != player) {
+            throw java.lang.Exception("Player ${player} cannot add army to country")
```

Comment:
Hace falta el package acá? Creo que se da cuenta java solo
```suggestion
            throw Exception("Player ${player} cannot add army to country")
```